### ISSUE TITLE
Hunter Pet abilities

### ISSUE THREAD

**Issue by Articy** [dalaranwow/dalaran-wow#4657]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. Spell bonus that the Hunter pets have doesn't stack with his abilities, making his spell damage amazingly low. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
> Spell bonus that the Hunter pets have doesn't stack with his abilities, making his spell damage amazingly low.

I don't fully understand what do you mean by that, however the damage of the pet's abilities is scaled with the hunter's attack power (the higher it is, the more the damage of the ability will be).
If @Articy would like to add sometime please comment below so if there is an issue we can identify it.
For now marking as:

**Comment by Edzero** [comment]
He's probably referring to the issues in #3774 which in turn is waiting for 3088 to be applied to PTR before it can be tested.
